### PR TITLE
Improve container frame handling

### LINKED ISSUE #20: Improve frame handling
Right now it doesn't support controllers that have that bars.

### PR DESCRIPTION
**Pull request by 3lvis**
This might not be ideal but I think is a step in the right direction, now the user will be able to define the container frame.

Fixes #20

### CODE REVIEW

File: Sources/SectionScrubber.swift [hunk 1]
```diff
@@ -125,17 +131,10 @@ public class SectionScrubber: UIView {
...
-        }
-
-        if let originalOriginY = self.originalOriginY {
-            self.containingViewFrame = CGRectMake(0, originalOriginY, self.collectionView.bounds.width, self.collectionView.bounds.height - originalOriginY - self.viewHeight)
-            self.containingViewContentSize = self.collectionView.contentSize
-            self.updateFrame() { _ in }
-        }
+        self.containingViewFrame = self.dataSource?.sectionScrubberContainerFrame(self) ?? CGRectZero
+        self.containingViewContentSize = self.collectionView.contentSize
+        self.updateFrame() { _ in }
```
Comment (elland) [inline Sources/SectionScrubber.swift:137]: 🤔 what's with the blocks? `{ _ in }`

Comment (3lvis) [inline Sources/SectionScrubber.swift:137]: @Elland it's an unused return.

**Comment by 3lvis** [comment]
at-work-cookie-monster

File: iOS/RemoteCollectionController.swift [hunk 2]
```diff
@@ -14,6 +14,7 @@ class RemoteCollectionController: UICollectionViewController {
     lazy var sectionScrubber: SectionScrubber = {
         let scrubber = SectionScrubber(collectionView: self.collectionView!)
         scrubber.delegate = self
+        scrubber.dataSource = self
```
Comment (marijnschilling) [inline iOS/RemoteCollectionController.swift:17]: Couldn't we just use the normal delegate to handle this function? I don't really see why it's a 'dataSource' it's just setting the frame right?

Comment (3lvis) [inline iOS/RemoteCollectionController.swift:17]: Delegate is used to inform. DataSource is used to provide.

Comment (marijnschilling) [inline iOS/RemoteCollectionController.swift:17]: Aha I guess that makes sense than! Good to know! 👍

**Comment by marijnschilling** [comment]
I like it! Might even be ideal, merrrrrge! :v:

File: Sources/SectionScrubber.swift (hunk 1 above)
Comment (elland) [inline Sources/SectionScrubber.swift:137]: Can't this be omitted?

Comment (3lvis) [inline Sources/SectionScrubber.swift:137]: I don't think so.

screen shot 2016-07-04 at 09 44 03

screen shot 2016-07-04 at 09 44 15

**Pull request merged**